### DISCUSSION TITLE
Feature Request: Changeable Windows 10 Start Menu Tile Colors

### DISCUSSION THREAD

**Discussion by rmz4ed** [valinet/ExplorerPatcher#2487] [Ideas]
In Windows 10, it was possible to change the start menu tile colors to different ones, but in the current EP Windows 10 start menu, they are all grey by default. It would be really nice if it was possible to change the colors like in Windows 10.
Current EP Tiles:
<img width="80" alt="Uncolored Tile" src="https://github.com/valinet/ExplorerPatcher/assets/74930865/bb90d34d-a82f-439b-b5ea-2be89d632407">
Old Windows 10 Tiles:
<img width="78" alt="Colored Tile" src="https://github.com/valinet/ExplorerPatcher/assets/74930865/a9b22365-f44c-4746-a827-92fac62eea0f">

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Just in case this was already added.

**Reply by rmz4ed** [reply]
Windows Version: 23H2
OS Build: 22631.2715
EP Version: 22621.2506.60.1

Using Windows 10 start menu setting with Show more tiles checked.

**Reply by pyrates999** [reply]
thank you.
